### PR TITLE
hw2 for check

### PR DESCRIPTION
**Pull request by demony**

### CODE REVIEW

File: src/main/java/dto/UserGenerator.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package dto;
+
+public class UserGenerator {
+    public static User getNewUser() {
+        return User
+            .builder()
+            .name("Robert")
```
Comment (CimaCha) [inline src/main/java/dto/UserGenerator.java]: Please move this hardcoded values to property file

File: src/test/java/scenarios/NativeMobileTests.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package scenarios;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import dto.User;
+import dto.UserGenerator;
+import java.util.List;
```
Comment (CimaCha) [inline src/test/java/scenarios/NativeMobileTests.java]: clean up exceeded imports

File: src/test/java/scenarios/NativeMobileTests.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+public class NativeMobileTests extends BaseTest {
+
+    @Test(groups = {"native"}, description = "This simple test just click on the Sign In button", enabled = true)
+    public void simpleNativeTest() throws IllegalAccessException, NoSuchFieldException, InstantiationException {
+        getPageObject().getWebElement("signInButton").click();
+        System.out.println("Simplest Android native test done");
+    }
+
+    @Test(groups = {"native"}, description = "Work with new account test - create and login", enabled = false)
```
Comment (CimaCha) [inline src/test/java/scenarios/NativeMobileTests.java]: enable this test back

File: src/test/java/scenarios/WebMobileTests.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                                        .equals("Internet Assigned Numbers Authority") : "This is not IANA homepage";
+
+        // Log that test finished
+        System.out.println("Site opening done");
+    }
+
+    @Test(groups = {"web"}, description = "Check that google search is relevant",
+    dataProvider = "searchStringsForGoogle", dataProviderClass = WebDataProvider.class)
+    public void googleSearchTest(String searchCase, String searchText, String relevantText) throws NoSuchFieldException, IllegalAccessException, InstantiationException {
+        getDriver().get("https://google.com");
```
Comment (CimaCha) [inline src/test/java/scenarios/WebMobileTests.java]: Please move hardcoded url to property file

File: src/main/java/dto/UserGenerator.java (hunk 1 above)
Comment (demony) [reply src/main/java/dto/UserGenerator.java]: fixed

File: src/test/java/scenarios/NativeMobileTests.java (hunk 3 above)
Comment (demony) [reply src/test/java/scenarios/NativeMobileTests.java]: fixed

File: src/test/java/scenarios/WebMobileTests.java (hunk 4 above)
Comment (demony) [reply src/test/java/scenarios/WebMobileTests.java]: fixed

**Pull request merged**